apps/survey/src/components/layouts/meal-list/list-mobile.vue

```diff
@@ -47,34 +61,70 @@ export default defineComponent({
 
   emits: ['action'],
 
-  setup() {
-    const { getLocaleContent } = useLocale();
-    const { getMealTime } = useMealUtils();
-
-    return { getLocaleContent, getMealTime };
-  },
-
   computed: {
```

Comment:
Can you extract shared logic of desktop and mobile version to composable and reuse for both?